pxd_fastpath.c

```diff
@@ -23,35 +23,131 @@
 // global fastpath IO work queue
 static struct workqueue_struct *gwq;
 
+#define MAX_PXFP_WORKERS_PER_NODE (2) /// keep it power of 2.
```

Comment:
Would it be helpful on systems where blk-mq is enabled to map this to the nr_hw_queues setting or mapping this to what ideal values should be for nr_hw_queues based on the backend media type?

Reply:
px uses a static configured 8 controller to fetch IOs on the coordinator side. This may be more involved, with benefits not obvious.